spec/features/plans_spec.rb

```diff
@@ -10,7 +10,8 @@
     @org = create(:org)
     @research_org = create(:org, :organisation, :research_institute,
                            name: 'Test Research Org', templates: 1)
-    @funding_org  = create(:org, :funder, name: 'Test Funder Org', templates: 1)
+    Rails.application.config.default_funder_name = 'Test Funder'
```

Comment:
After merging #814, this will have to use Rails.application.config.default_funder_id